lib/ios-deploy.js

```diff
@@ -19,7 +19,7 @@ class IOSDeploy {
   async remove (bundleid) {
     let remove = [`--uninstall_only`, `--id`, this.udid, `--bundle_id`, bundleid];
     try {
-      await exec(this.cmd, remove, { maxBuffer: 524288});
+      await exec(this.cmd, remove, {maxBuffer: 524288});
```

Comment:
I know this code was here before, but shall we take the opportunity to remove `maxBuffer` options throughout this file since `teen_process` handles that on its own now?